eclair-core/src/test/scala/fr/acinq/eclair/StartupSpec.scala

```diff
@@ -211,15 +211,7 @@ class StartupSpec extends AnyFunSuite {
       """.stripMargin
     )
 
-    val nodeParams = makeNodeParamsWithDefaults(perNodeConf.withFallback(defaultConf))
-    val perNodeFeaturesA = nodeParams.initFeaturesFor(PublicKey(ByteVector.fromValidHex("02aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")))
-    val perNodeFeaturesB = nodeParams.initFeaturesFor(PublicKey(ByteVector.fromValidHex("02bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb")))
-    val defaultNodeFeatures = nodeParams.initFeaturesFor(PublicKey(ByteVector.fromValidHex("02cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc")))
-    // Some features should never be sent in init messages.
-    assert(nodeParams.features.hasFeature(PaymentMetadata))
-    assert(!perNodeFeaturesA.hasFeature(PaymentMetadata))
-    assert(!perNodeFeaturesB.hasFeature(PaymentMetadata))
-    assert(!defaultNodeFeatures.hasFeature(PaymentMetadata))
+    assertThrows[IllegalArgumentException](makeNodeParamsWithDefaults(perNodeConf.withFallback(defaultConf)))
```

Comment:
The test should be renamed ("reject non-init features in node override")

Reply:
Done